src/com/epam/Member.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.training;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+public class Member {
+    public Scanner input = new Scanner(System.in);
+    public Scanner input1 = new Scanner(System.in);
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
+    LocalDateTime now = LocalDateTime.now();
+    private String name;
+    public int action;
```

Comment:
in my understanding you store last message and last action for each member., so rename fields to correspond the logic, like lastAction, lastMessage